File: Modul-01/python/PRAK105-2310817210009-MuhammadIbnuSina.py
# Memasukkan input dari pengguna untuk variabel a, b, x, dan y
a = float(input("Variabel a bernilai: "))
b = float(input("Variabel b bernilai: "))
x = float(input("Variabel x bernilai: "))
y = float(input("Variabel y bernilai: "))

# Menghitung total sisa bagi dari a dibagi b dan x dibagi y
sisa_bagi_a = a % b
sisa_bagi_x = x % y

# hasil
print(f"Total sisa bagi dari a dibagi b dan x dibagi y adalah {int(sisa_bagi_a + sisa_bagi_x)}")

File: Modul-01/c/PRAK105-2310817210009-MuhammadIbnuSina.c
#include <stdio.h>

int main() {
    float a, b, x, y;

    printf("Variabel a bernilai: ");
    scanf("%f", &a);

    printf("Variabel b bernilai: ");
    scanf("%f", &b);

    printf("Variabel x bernilai: ");
    scanf("%f", &x);

    printf("Variabel y bernilai: ");
    scanf("%f", &y);

    float sisa_a_b = a - ((int)(a / b) * b);
    float sisa_x_y = x - ((int)(x / y) * y);

    printf("Total sisa bagi dari a dibagi b dan x dibagi y adalah %d\n", (int)(sisa_a_b + sisa_x_y));

    return 0;
}
